### ISSUE TITLE
How to build a project like https://www.markmind.org/

### ISSUE THREAD

**Issue by jiaoxicai2013** [MarkMindLtd/Mark-Mind#19]
How to build a project like https://www.markmind.org/

**Comment by MarkMindLtd** [comment]
Do you want to deploy on your own server ？

**Comment by jiaoxicai2013** [comment]
Yes,I want to build project on my server. Thanks for your help if you can~

**Comment by jiaoxicai2013** [comment]
> Do you want to deploy on your own server ？

Yes,I want to build project on my server. Thanks for your help if you can~
